ports/esp32/machine_uart.c

```diff
@@ -59,6 +59,7 @@
 #define UART_IRQ_BREAK (1 << UART_BREAK)
 #define MP_UART_ALLOWED_FLAGS (UART_IRQ_RX | UART_IRQ_RXIDLE | UART_IRQ_BREAK)
 #define RXIDLE_TIMER_MIN (5000)  // 500 us
+#define UART_QUEUE_SIZE (3)
```

Comment:
This number 3 is the exact same magic number that was previously used when IRQ RXIDLE support was introduced in #14041. I only moved it to a define since it's now used in two places. I don't know why it's 3 and didn't see any mention in the previous MR about that, but decided to not change it since it seems to work fine.